components/filter-header/filter-header.html

```diff
@@ -38,21 +38,12 @@
         cursor: pointer;
     }
 
-    #btnClose {
-        display: var(--close-display);
+    .icon{
+        font-family: crsfrw;
+        font-size: 1.5rem;
```

Comment:
check with chuck

Reply:
Does not have to be class icon, as long as font-family: crsfrw is there.